packages/react-native-web/src/exports/DeviceInfo/index.js

```diff
@@ -34,7 +34,7 @@ const DeviceInfo = {
     }
   },
 
-  get locale() {
+  get locale(): any | void {
```

Comment:
Shouldn't `any` be `string` in this case?

Reply:
Yeah these any types are from the codemod, and it turns out flow doesn't have navigator fully typed